.github/workflows/cups-build-and-publish.yml

```diff
@@ -3,7 +3,7 @@ name: Docker CUPS-debian build and publish
 on: workflow_dispatch
 
 env:
-  IMAGE_NAME: sanger/cupsd
+  IMAGE_NAME: ${{ github.repository }}/cupsd
```

Comment:
This will be sanger/print_my_barcode/cupsd, right?

Reply:
Yep. There is an example of image built here: https://github.com/sanger/print_my_barcode/pkgs/container/print_my_barcode%2Fcupsd